experimental/mrpc/plugin/policy/delay/src/config.rs

```diff
@@ -0,0 +1,37 @@
+use chrono::{Datelike, Timelike, Utc};
+use phoenix_common::log;
+use serde::{Deserialize, Serialize};
+
+use chrono::prelude::*;
+use itertools::iproduct;
+use rand::Rng;
+
+#[derive(Debug, Clone, Copy, Default, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct DelayConfig {}
+
+impl DelayConfig {
+    /// Get config from toml file
+    pub fn new(config: Option<&str>) -> anyhow::Result<Self> {
+        let config = toml::from_str(config.unwrap_or(""))?;
+        Ok(config)
+    }
+}
+
+pub fn create_log_file() -> std::fs::File {
```

Comment:
There are a few unused function such as `create_log_file` and `materialize_nocopy`. You can remove them.